app/packages/core/src/components/ColorModal/ColorFooter.tsx

```diff
@@ -89,7 +98,10 @@ const ColorFooter: React.FC = () => {
               setDatasetColorScheme({
                 variables: { subscription, datasetName, colorScheme: null },
               });
-              setColorScheme({ fields: [], colorPool: defaultColorPool });
+              setColorScheme({
+                fields: [],
+                colorPool: useRecoilValue(fos.config).colorPool,
```

Comment:
This hook is conditionally run. I think the config is already available  via `configDefault`

Reply:
Good catch. Thank you!